tests/api_test.py

```diff
@@ -9013,5 +9014,19 @@ def test_call_wrapped_second_phase_cleanup(self):
     assert core.trace_state_clean()
 
 
+class EnvironmentInfoTest(jtu.JaxTestCase):
```

Comment:
Can you add a test where `XLA_FLAGS=--xla_force_host_platform_device_count=256` and then run this and check if the list of devices is infact truncated?

I don't see where you are truncating it.

Reply:
I don't think you can change that flag value for a single test, can you?

Reply:
I added an example of truncated output to the CL description.